certbot/plugins/dns_common_lexicon.py

```diff
@@ -83,7 +83,7 @@ def _find_domain_id(self, domain):
                     self.provider.options['domain'] = domain_name
                 else:
                     # For Lexicon 3.x
-                    self.provider.domain = domain_name
+                    self.provider.get_domain = domain_name
```

Comment:
You probably didn't mean to also change this?

Reply:
Ah, no. Side effect of automated refactoring.

Reply:
Done.